### PR TITLE
New: Network details display

### PR DESCRIPTION
**Pull request by DL6ER**
**By submitting this pull request, I confirm the following:** 

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/AdminLTE/blob/master/CONTRIBUTING.md), as well as this entire template.
- [X] I have made only one major change in my proposed changes.
- [X] I have commented my proposed changes within the code.
- [X] I have tested my proposed changes.
- [X] I am willing to help maintain this change if there are issues with it later.
- [X] I give this submission freely and claim no ownership.
- [X] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [X] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))
- [X] I have Signed Off all commits. ([`git commit --signoff`](https://git-scm.com/docs/git-commit#git-commit---signoff))

---

**What does this PR aim to accomplish?:**

Add a frontend for the new information generated by FTL in https://github.com/pi-hole/FTL/pull/455

screenshot from 2019-01-06 20-21-59

**How does this PR accomplish the above?:**

Add a new page `network.php` + a new (simple) DB-API callback `api_db.php` 

This PR also updated `font-awesome` from 4.5.0 to 5.6.3

**What documentation changes (if any) are needed to support this PR?:**

Nothing urgent.

### CODE REVIEW

File: scripts/pi-hole/php/header.php [hunk 1]
```diff
@@ -424,145 +424,151 @@ function pidofFTL()
                 <!-- Home Page -->
                 <li<?php if($scriptname === "index.php"){ ?> class="active"<?php } ?>>
                     <a href="index.php">
-                        <i class="fa fa-home"></i> <span>Dashboard</span>
+                        <i class="fa fas fa-home"></i> <span>Dashboard</span>
```
Comment (Mcat12) [inline scripts/pi-hole/php/header.php]: `fa` is an alias for `fas`, correct? Then only one is needed.

File: scripts/pi-hole/php/header.php [hunk 2]
```diff
@@ -582,21 +588,21 @@ function pidofFTL()
...
-                        <i class="fa fa-user"></i> <span>Login</span>
+                        <i class="fa far fa-user"></i> <span>Login</span>
                     </a>
                 </li>
                 <?php } ?>
                 <!-- Donate -->
                 <li>
                     <a href="https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=3J2L3Z4DHW9UY" target="_blank">
-                        <i class="fa fa-paypal"></i> <span>Donate</span>
+                        <i class="fa-icon fab fa-paypal"></i> <span>Donate</span>
```
Comment (Mcat12) [inline scripts/pi-hole/php/header.php]: The icon is not centered correctly:
image

**Review by AzureMarker — DISMISSED** [review]

File: scripts/pi-hole/php/header.php (hunk 1 above)
Comment (DL6ER) [reply scripts/pi-hole/php/header.php]: I know the documentation says they are identical, but they are not.

`class="fa fas fa-list-ul"` (as expected):
screenshot from 2019-01-07 18-54-02

`class="fas fa-list-ul"` (justification/width lost):
screenshot from 2019-01-07 18-54-18

It took me a while to get the things back into shape after updating `font-awesome`...

Comment (Mcat12) [reply scripts/pi-hole/php/header.php]: Is there a similar issue if you only use `fa`?

Comment (DL6ER) [reply scripts/pi-hole/php/header.php]: This works fine.

**Review by AzureMarker — APPROVED** [review]

**Pull request merged**